app/javascript/app/pages/regions/regions-ghg-emissions/regions-ghg-emissions-styles.scss

```diff
@@ -41,3 +40,13 @@
 .projectedLegend {
   margin-left: 0;
 }
+
+.targetChartsContainer {
+  @include columns($columns: 12);
+
+  margin-top: 50px;
+
+  @media #{$tablet-landscape} {
+    @include columns($columns: (6,6));
```

Comment:
this can be simplified to `@include columns(6)`

Reply:
hmm, it does not work

Reply:
I meant `columns(6,6)`, sorry

Reply:
Ok, if the column width is the same then I will use shorter `@include columns(6)`

Reply:
Thanks! :+1:

Reply:
🙇‍♀️